docs/setup-android.md

```diff
@@ -0,0 +1,46 @@
+# Setting up for Android
+For non-android modules in your project you can follow the [JVM](setup-jvm.md) instructions.
+
+```groovy
+// junit5 doesn't support android projects out of the box
```

Comment:
nit: comments should be sentences. https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html